src/GenerateEuler.m

```diff
@@ -0,0 +1,530 @@
+function GenerateEuler()

```

Comment:
Eek, what can I say, it's matlab :-/

Regardless of that unfortunate fact, let's put GenerateEuler.m somewhere other than src, since it's not actually part of the package source per se.  perhaps `gen/GenerateEuler.m`?